src/init.cpp

```diff
@@ -1022,6 +1023,16 @@ bool AppInitParameterInteraction(const ArgsManager& args)
         }
     }
 
+    if (args.IsArgSet("-blockreservedweight")) {
+        const auto block_reserved_weight = args.GetIntArg("-blockreservedweight", DEFAULT_BLOCK_RESERVED_WEIGHT);
```

Comment:
In cd6cca1018f1c4b0c5d73991afbdb19d1b0b9682 "miner: init: add `-blockreservedweight` startup option"

Same as for the earlier commit, imo this should only check against the consensus limit.
https://github.com/bitcoin/bitcoin/pull/31384/commits/cd6cca1018f1c4b0c5d73991afbdb19d1b0b9682#r1938237527

Reply:
Done!